plugins/module_utils/cmd_runner_fmt.py

```diff
@@ -126,5 +140,5 @@ def stacking(value):
     return wrapper
 
 
-def is_argformat(fmt):
+def is_argformat(fmt: object) -> t.TypeGuard[_ArgFormat]:
```

Comment:
?
I would have thought this to be
```suggestion
def is_argformat(fmt: object) -> bool:
```

Reply:
A type guard means that the return value is a boolean, which has the meaning that the argument to the function is of the specified type. I.e. if `is_argformat(x)` returns `True`, then `x` is of type `_ArgFormat`. So if you have `if is_argformat(x): ...` the type checker knows that inside the `if'`s body, `x` is actually of type `_ArgFormat`.